DirectEmployers/MyJobs
# [PD-1627] Create Topbar Widget for Secure Blocks

## Overview

This ticket involves the creation of a topbar widget (renamed ToolsWidget) that will use the secure blocks API for retrieval. This widget operates in the very similar fashion to the existing topbar widget, but has CORS capability and customization options.

## Testing

Like all of the other secure blocks PRs, this requires a working https overwrite such as done with Docker

* Head to admin
* Create a new ToolsWidget (under MyBlocks)
 * Name it 'tools-widget'. 0 for span/offset, template default or modify it a bit to make it distinguishable from the normal topbar widget
* Make sure my.jobs' configuration is using the homepage template of CS_TEST_homepage_listing.html
* Navigate to secure.my.jobs.. Log in (this is required unless you remove the staff restriction currently on secure-blocks)
* Navigate to my.jobs.. Topbar should appear (with any modifications to ensure you're seeing the correct topbar
